tvtk/wrapper_gen.py

```diff
@@ -620,6 +620,22 @@ def _gen_get_set_methods(self, klass, out):
                     t_def += 'enter_set=True, auto_set=False)'
                     self._write_trait(out, name, t_def, vtk_set_meth,
                                       mapped=False)
+                elif PY_VER < 3 and typ is unicode:
+                    default = default.encode("unicode_escape")
+                    if '\n' in default or '\r' in default:
+                        default = clean_special_chars(default)
+
+                    if default == '\x00':
+                        t_def = 'traits.Unicode(u"", '
+                    elif default == '"':
+                        t_def = "traits.Unicode(u'%(default)s', " % locals()
```

Comment:
@rkern done. I also took care of the part handling String and removed the cleaning function.